addons/sys_intercom/fnc_handleAccent.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Handles setting up accent in the master station.
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 2: Intercom network <NUMBER>
+ * 3: Accent active <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [(vehicle acre_player), "intercom1", true] call acre_sys_intercom_fnc_handleAccent
+ *
+ * Public: No
+ */
+
+params ["_vehicle", "_intercomNetwork", "_accentActive"];
```

Comment:
Any defaults needed here? Or are we sure nothing is accidentally calling this function?

Reply:
Default values are set during intercom configuration. Since this is an internal function I did not plan any. It is only called using the ACE menu.